apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -39,3 +46,23 @@ type Answer struct {
 	ClusterName string            `json:"clusterName,omitempty" norman:"type=reference[cluster]"`
 	Values      map[string]string `json:"values,omitempty" norman:"required"`
 }
+
+type MultiClusterAppRevision struct {
+	types.Namespaced
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	TemplateVersionName string   `json:"templateVersionName,omitempty" norman:"type=reference[templateVersion],required"`
+	Answers             []Answer `json:"answers,omitempty"`
+}
+
+type MultiClusterAppUpgradeInput struct {
+	TemplateVersionName string   `json:"templateVersionName,omitempty"`
+	Answers             []Answer `json:"answers,omitempty"`
+	BatchSize           int      `json:"batchSize,omitempty"`
```

Comment:
Also we might want to set the strategy as the field on the MCA, instead of setting it as an action input. Similar to how k8s does. This way you don't have to pass it on upgrade and rollback actions